resources/configurations/examples/wan_client.yaml

```diff
@@ -74,25 +66,23 @@ WANClient:                                                        # 6
 # 8: Set the Discovery Server GuidPrefix to <01.0f.02.00.00.00.00.00.00.00.ca.fe> for WANClient.
 
 # 9: Add the addresses where to reach the remote DDS Routers that will connect to.
+# Add as many connection-addresses as needed.
 
 # 10: Set the Remote Discovery Server GuidPrefix to <01.0f.01.00.00.00.00.00.00.00.ca.fe>.
 
 # 11: Connect to a Discovery Server in IP <1.1.1.1> listening in port 11666.
```

Comment:
Remove UDP type and say it is by default, or comment that line as a new point

Reply:
:+1: